--- src/agg/process_line_pattern_symbolizer.cpp	2018-06-09 18:58:11.999537449 +0200
+++ ../process_line_pattern_symbolizer.pathed	2018-06-09 19:06:55.693741817 +0200
@@ -121,7 +121,7 @@
         pattern_type pattern (filter,source);
         renderer_type ren(ren_base, pattern);
         double half_stroke = std::max(width / 2.0, height / 2.0);
-        int rast_clip_padding = static_cast<int>(std::round(half_stroke));
+        int rast_clip_padding = static_cast<int>(round(half_stroke));
         ren.clip_box(-rast_clip_padding,-rast_clip_padding,common_.width_+rast_clip_padding,common_.height_+rast_clip_padding);
         rasterizer_type ras(ren);
 

