--- include/mapnik/css_color_grammar_x3_def.hpp	2018-06-09 18:58:11.879538831 +0200
+++ ../css_color_grammar_x3_def.patched	2018-06-09 19:02:46.612443647 +0200
@@ -244,7 +244,7 @@
 {
     static std::uint8_t call(double val)
     {
-        return safe_cast<std::uint8_t>(std::lround((255.0 * val)/100.0));
+        return safe_cast<std::uint8_t>(lround((255.0 * val)/100.0));
     }
 };
 
@@ -348,9 +348,9 @@
     double g = hue_to_rgb(m1, m2, h);
     double b = hue_to_rgb(m1, m2, h - 1.0/3.0);
     uint8_t alpha = uint8_t((255.0 * clip_opacity::call(std::get<3>(_attr(ctx)))) + 0.5);
-    _val(ctx) = color(safe_cast<uint8_t>(std::lround(255.0 * r)),
-                      safe_cast<uint8_t>(std::lround(255.0 * g)),
-                      safe_cast<uint8_t>(std::lround(255.0 * b)),
+    _val(ctx) = color(safe_cast<uint8_t>(lround(255.0 * r)),
+                      safe_cast<uint8_t>(lround(255.0 * g)),
+                      safe_cast<uint8_t>(lround(255.0 * b)),
                       alpha);
 };
 
